### PR TITLE
first html-css-popup commit

### PR DESCRIPTION
**Pull request by eve5ince**
### HTML CSS Popup 

[Demo](https://eve5ince.github.io/html-css-popup/)
[Code base](https://github.com/eve5ince/html-css-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Hi! Nice job, but some things must be redone.
Снимок экрана 2020-11-06 в 10 55 29 PM

File: submissions/eve5ince/html-css-popup/css/master.css [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+body {
+	background: rgb(238,210,232);
+	background: linear-gradient(45deg, rgba(238,210,232,0.7889355571330094) 22%, rgba(99,252,69,0) 100%); 
+	font-family: Arial, Helvetica, sans-serif;
+}
+nav>ul {
+   display: flex;
+   justify-content: space-between;
+	margin-top: 30px;
+	margin-left: 900px;
```
Comment (madmaxWMFU) [inline submissions/eve5ince/html-css-popup/css/master.css:18]: Use name of class for more informative.
Delete rule `margin-top: 30px;` and `margin-left: 900px;`
Set value `flex-end` for `justify-content`.

File: submissions/eve5ince/html-css-popup/css/master.css [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+.links>a:hover, .more>a:hover{
+	outline: 1px solid silver;
+}
+.links>a:focus, .more>a:focus {
+	 background: rgba(238,210,232, 0.2); 
+}
+.links>img>p {
+	font-size: 14px;
+}
+li:first-of-type {
```
Comment (madmaxWMFU) [inline submissions/eve5ince/html-css-popup/css/master.css:86]: Try `li + li { padding: 0 15px; }` instead `li:first-of-type` and `li:last-of-type`

File: submissions/eve5ince/html-css-popup/css/master.css [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+	padding-left: 30px;
+}
+li:last-of-type {
+	padding-right: 30px;
+}
+input[type="checkbox"] {
+	display: none;
```
Comment (madmaxWMFU) [inline submissions/eve5ince/html-css-popup/css/master.css:93]: When you use this property the button does not work when a space is pressed.
Better to use `opacity: 0;`
Don't forget about focus on this element `input[type="checkbox"]:focus + .nav-img {....}`

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Good job!
See some comments below

File: submissions/eve5ince/html-css-popup/index.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+	<nav>
+		<ul>
+			<li><a href="#" class="">Mail</a></li>
+			<li><a href="#" class="">Images</a></li>
+			<li>
+				<label for="popUp" class="nav-img popup-button" tabindex="0"></label>
+				<input type="checkbox" id="popUp">
+				<div class="control">
+					<div class="links">
+						<a href="#"><img src="img/calendar.png" class="img"><p>Calendar</p></a>
```
[the remark this answers is not in the document]
Comment (eve5ince) [reply submissions/eve5ince/html-css-popup/index.html]: Thank you for reviewing!
I've made all the changes.

File: submissions/eve5ince/html-css-popup/css/master.css (hunk 1 above)
Comment (eve5ince) [reply submissions/eve5ince/html-css-popup/css/master.css]: Thank you kindly for reviewing!

File: submissions/eve5ince/html-css-popup/css/master.css (hunk 3 above)
Comment (eve5ince) [reply submissions/eve5ince/html-css-popup/css/master.css]: Thx, this was helpful.

File: submissions/eve5ince/html-css-popup/css/master.css [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+	margin-top: 30px;
+	margin-left: 900px;
+	align-items: center;
+	font-size: 18px;
+	background-color: transparent;
+	padding: 5px 0 5px 0;
+	border-radius: 25px;
+	letter-spacing: 0.05em;
+}
+nav>ul>li>a:hover{
```
[the remark this answers is not in the document]
Comment (eve5ince) [reply submissions/eve5ince/html-css-popup/css/master.css]: Yes, sir!

File: submissions/eve5ince/html-css-popup/css/master.css (hunk 2 above)
Comment (eve5ince) [reply submissions/eve5ince/html-css-popup/css/master.css]: Thx, I ended simply with adding margins.

**Comment by madmaxWMFU** [comment]
Please update your demo also to see changes

File: submissions/eve5ince/html-css-popup/index.html [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  						</div>
+  					</details> 	
+				</div>
+			</li>
+			<li><a href="#" class="nav-img bell"></a></li>
+			<li><a href="#" class="nav-img profile"></a></li>
+		</ul>			
+	</nav>
+</body>
+</html>
```
[the remark this answers is not in the document]
Comment (eve5ince) [reply submissions/eve5ince/html-css-popup/index.html]: Thx, added.

**Comment by eve5ince** [comment]
@madmaxWMFU suure, thank you for reminding, demo is upragded now

File: submissions/eve5ince/html-css-popup/css/master.css [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+	height: 28px;
+}
+a:focus, input:focus {
+  outline: 1px solid #757575;
+}
+.bell {
+	width: 28px;
+	height: 28px;
+}
+#popUp {
```
Comment (madmaxWMFU) [inline submissions/eve5ince/html-css-popup/css/master.css:98]: It is not necessary, you have already applied the value for `lable`

File: submissions/eve5ince/html-css-popup/css/master.css [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+	margin-top: 20px;
+}
+.more {
+	margin-bottom: 20px;
+}
+.img {
+	display: inline-block;
+	margin: 10px 0;
+	background-repeat: no-repeat;
+	background-size: contain;
```
Comment (madmaxWMFU) [inline submissions/eve5ince/html-css-popup/css/master.css:114]: Pictures will not be stretched using this property `object-fit: contain;`

File: submissions/eve5ince/html-css-popup/index.html [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+						<a href="#" class="links-item"><img src="img/files.png" class="img" alt="Files"><p>Files</p></a>
+						<a href="#" class="links-item"><img src="img/contacts.png" class="img" alt="Contacts"><p>Contacts</p></a>
+						<a href="#" class="links-item"><img src="img/tanslation.png" class="img" alt="Translation"><p>Translate</p></a>
+						<a href="#" class="links-item"><img src="img/videos.png" class="img" alt="Videos"><p>Video</p></a>
+						<a href="#" class="links-item"><img src="img/search.png" class="img" alt="Search"><p>Search</p></a>
+						<a href="#" class="links-item"><img src="img/photos.png" class="img" alt="Photos"><p>Photo</p></a>
+						<a href="#" class="links-item"><img src="img/apps.png" class="img" alt="Apps"><p>Apps</p></a>
+						<a href="#" class="links-item"><img src="img/documents.png" class="img" alt="Documents"><p>Document</p></a>
+					</div>
+					<details>
```
Comment (madmaxWMFU) [inline submissions/eve5ince/html-css-popup/index.html:32]: This block must be invisible after a click

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Avoid naming by id and tag name

File: submissions/eve5ince/html-css-popup/css/master.css (hunk 7 above)
Comment (eve5ince) [reply submissions/eve5ince/html-css-popup/css/master.css:98]: Done

**Comment by madmaxWMFU** [comment]
Add a focus on the button more and you get approval.
`.extra__input:focus ~ .extra__label {....}`

**Review by madmaxWMFU — APPROVED** [review]
LGTM!

**Comment by eve5ince** [comment]
@madmaxWMFU finally, thank you kidnly for reviewing!

**Review by miroslav-kolomiets — APPROVED** [review]
Good job! Ready for merge)

**Pull request merged**